pyviz_comms/__init__.py

```diff
@@ -0,0 +1,435 @@
+import json
+import uuid
+import sys
+import traceback
+
+try:
+    from StringIO import StringIO
+except:
+    from io import StringIO
+
+
+
+PYVIZ_PROXY = """
+if (window.PyViz === undefined) {
+   if (window.HoloViews === undefined) {
+     var PyViz = {comms: {}, comm_status:{}, kernels:{}, receivers: {}, plot_index: []}
```

Comment:
Consistent spacing would be nice if you're already making the change above.

Reply:
Fixed in e9f2157